packages/story-editor/src/components/styleManager/styleManager.js

```diff
@@ -73,7 +73,7 @@ const StyledText = styled(Text).attrs({
   color: ${({ theme }) => theme.colors.fg.primary};
 `;
 
-function StyleManager({ styles, onClose, applyStyle }) {
+function StyleManager({ styles, onClose, applyStyle, ...rest }) {
```

Comment:
This dialog seems to lack focus trapping.

Reply:
Also, when you delete an element using the keyboard inside the dialog, focus is lost and returned to the selected element on canvas rather than staying inside the dialog. But both of these issues probably existed before, so could be moved to follow-up issues.